hugo/content/contribute.md

```diff
@@ -3,83 +3,239 @@ title: Contribute
 
-Please vote with a thumbs up on those you would like to see being fulfilled, describe your use cases or otherwise participate in the discussion on existing tickets. And if you want to request a feature not suggested yet, please create a new feature request ticket possibly with some elaboration and use cases.
+-->
 
-# Test new features and developments
+#### Open issues and ongoing development
 
-Apart from our stable releases we also provide development snapshots for testing the current, intermediate state of development. You can download them from our [Downloads page]({{< relref "downloads" >}})).
+Take a look at open [issues](https://github.com/mumble-voip/mumble/issues) and [pull requests](https://github.com/mumble-voip/mumble/pulls) on GitHub, especially with the following Labels:
 
-If you find any issues with them please check if the issue has already been reported, add details if you have more, or add a new tracker item if it has not been reported yet.
+<!-- add label for important tasks? -->
 
-# Translation
+- [Good first issue](https://github.com/mumble-voip/mumble/labels/good%20first%20issue)
+- [Help wanted](https://github.com/mumble-voip/mumble/labels/help%20wanted)
+- [Help needed](https://github.com/mumble-voip/mumble/labels/help-needed)
+- [Feature Request](https://github.com/mumble-voip/mumble/labels/feature-request)
+- [Bug](https://github.com/mumble-voip/mumble/labels/bug)
+
+
+#### Tips
+
+- To avoid unnecessary work and duplication, comment on issues and inform others what you plan to work on.
+- You can also create [Draft Pull Requests](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-pull-requests#draft-pull-requests); this way you can work on the Pull Request, while others can see the progress and give you feedback.
+
+### Additional projects
+
+#### 3rd party Interfaces
+
+Web interfaces, web-viewers, administration programs, CMS website modules – all neat things we would love to see! These could be written in any language supported by our Ice based or gRPC based RPC Interfaces. You could start your own project or link up with an {{< wiki "3rd Party Applications" "already existing one" />}}. To get an idea about how interaction with the Mumble server might look like, you can take a look at the example scripts in our [GitHub repository script folder](https://github.com/mumble-voip/mumble/tree/master/scripts).
+
+#### Ice authenticators
+
+See {{< wiki Authenticators />}}. We’re happy for every authenticator and authenticator improvement.
+
+#### Positional Audio plugins
+
+Most of our positional audio plugins regularly break on game updates. To read on how you can help us update the plugins see the {{< wiki Pluginguide />}} page. 
+You can also add native positional audio support to an Open Source game (or a game where you are allowed to edit the source code (for example if you are a developer)), see {{< wiki "Link" "Linking a game to Mumble" />}}.
```

Comment:
Is this correct? @Kissaki

Reply:
That you can use the Link plugin for that? Yes that is correct

Reply:
:thinking: To be more precise: So the formulation of the second sentence is correct?
- add native positional audio support to a game with that Link code?
- and how does that work then; without any plugin?

Reply:
`Link` is actually the plugin that is used for that. It works by providing an area of shared memory into which some endpoint on the game's size continuously writes the position information. And the Link plugin then fetches this information from the shared memory in order to send it to Mumble.

Reply:
Ok, thanks for the info. So there is no need for additional plugins in this case?